### DISCUSSION TITLE
Build Server for Docker Compose apps

### DISCUSSION THREAD

**Discussion by JasonDevTech** [coollabsio/coolify#3221] [Feature Requests]
Currently, build servers cannot be enabled for docker compose apps as seen here: https://github.com/coollabsio/coolify/blob/69fc4c7f525100c9291b75f4b9f31f92b285bf14/resources/views/livewire/project/application/general.blade.php#L229-L236, but the expectation is that build server is available for all build pack types.

**Comment by takayumi** [comment]
Since Coolify doesn't support replicas for containers or Nixpacks, I had to resort to Docker Compose for building my services.

Just found out that external Build Servers don't work and I'm pretty disappointed.

Are you still using Coolify or did you opt in to create your own custom CI/CD solution, @JasonDevTech?

**Comment by zaosoula** [comment]
+1

**Comment by tristanschwenk** [comment]
+1

**Comment by johandenhollander** [comment]
Came here searching for answers. Had done a lot of builds on a new server until it ran out of diskspace. Was under the impression that Having a build server would make all builds be build there.
Lesson learned today. docker-compose via Coolify can't use the build server. The butedild server docs don't mention this.

For now activated the cleanup options on the server. Should work as well. But doing builds will give some performance impact on a running system now then.

**Comment by fenicento** [comment]
+1
